src/pages/Login/Login.tsx

```diff
@@ -2,6 +2,7 @@ import { useTranslation } from 'react-i18next';
 import { Page } from '../../core/components/Page';
 import { LoginForm } from '../../user/components/LoginForm';
 import './LoginPage.css';
+import { LanguageSwitcherContainer } from '../../core/components/LanguageSwitcher/LanguageSwitcherContainer';
```

Comment:
Container is a kind of system component (controller, wrapper). At the end, we want to use a component by itself. And we dont care if it is a wrapper or container or pure presentational component. So, if we have containers and components in the same folder (that should have the name same as the component), lets re-export containers with the name of component. It makes cleaner code and points the usage of component

To do so, please add an index.ts file to the` ../../core/components/LanguageSwitcher ` folder, and make re-export:
```typescript
export { LanguageSwitcherContainer as LanguageSwitcher } from './LanguageSwitcherContainer';
```
Then import can be simplified:
````typescript
import { LanguageSwitcher } from '../../core/components/LanguageSwitcher';

Reply:
I changed the name in index file. Seems to be working ok.